src/test/java/io/vertx/test/core/HttpTest.java

```diff
@@ -3767,6 +3769,63 @@ public void testFollowRedirectHost() throws Exception {
     await();
   }
 
+  @Test
+  public void testSetExceptionHandler() throws Exception {
```

Comment:
it would be better to test this without casting and instead trigger an exception on the stream, like the server closes the stream during a response transfer

Reply:
Trying to find an example on how to do it...

Reply:
@vietj sorry, I cannot figure out how to get raise that exception on the stream. Would you mind pointing me to what you are talking about?